pkg/daemon/rpm-ostree.go

```diff
@@ -236,14 +236,23 @@ func (r *RpmOstreeClient) IsNewEnoughForLayering() (bool, error) {
 	return false, nil
 }
 
-// RebaseLayered rebases system or errors if already rebased
+// RebaseLayered rebases system or errors if already rebased.
 func (r *RpmOstreeClient) RebaseLayered(imgURL string) (err error) {
 	// Try to re-link the merged pull secrets if they exist, since it could have been populated without a daemon reboot
 	useMergedPullSecrets()
 	klog.Infof("Executing rebase to %s", imgURL)
 	return runRpmOstree("rebase", "--experimental", "ostree-unverified-registry:"+imgURL)
 }
 
+// RebaseLayeredFromContainerStorage rebases the system from an existing local container storage image.
+func (r *RpmOstreeClient) RebaseLayeredFromContainerStorage(imgURL string) (err error) {
+	// Try to re-link the merged pull secrets if they exist, since it could have been populated without a daemon reboot
+	useMergedPullSecrets()
+	klog.Infof("Executing local container storage rebase to %s", imgURL)
+	// just testing
```

Comment:
nit: this comment looks like was part of testing. should this we removed?

Reply:
yep will remove